app/controllers/scheduled_messages_controller.rb

```diff
@@ -18,7 +18,8 @@ def create
   end
 
   def index
-    @scheduled_messages = ScheduledMessage.filter(params).order(:days_after_start).page(params[:page])
+    @scheduled_messages = ScheduledMessage.filter(params)
```

Comment:
Oops, yes, not sure how that happened. Fixed in 5aa8be4c2f0bb5e3ee419081d7bd5b92eb4d3f00